### PR TITLE
Updated method for running 200 node test

### PR DESCRIPTION
**Pull request by sergio-mena**
Contributes to cometbft/qa-infra#2

Adapted the text with the new functionality, and the new uses cases we're doing now.

---

#### PR checklist

- [x] Tests written/updated, or no tests needed
- [x] `CHANGELOG_PENDING.md` updated, or no changelog entry needed
- [x] Updated relevant documentation (`docs/`) and code comments, or no
      documentation updates needed

### CODE REVIEW

File: docs/qa/method.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ _200 Node Test_, and _Rotating Nodes Test_.
     * [Terraform CLI][Terraform]
     * [Ansible CLI][Ansible]
 
-[testnet-repo]: https://github.com/interchainio/tendermint-testnet
+[testnet-repo]: https://github.com/interchainio/tendermint-testnet   FIX
```
Comment (jmalicevic) [inline docs/qa/method.md:32]: Is this  ToDo? Or we should leave it in to remember to fix it?

**Review by jmalicevic — APPROVED** [review]

File: docs/qa/method.md (hunk 1 above)
Comment (sergio-mena) [reply docs/qa/method.md:32]: Wow, didn't even see it :-(
Bear with me while I fix it

**Review by jmalicevic — APPROVED** [review]
All good!

**Review by lasarojc — APPROVED** [review]

File: docs/qa/method.md [hunk 2]
```diff
@@ -53,16 +53,36 @@ This section explains how the tests were carried out for reproducibility purpose
...
+        * Before running it, you need to edit the script to provide the IP address of a full node.
+          This node will receive all transactions from the load runner node.
+        * This script will take about 40 mins to run.
+        * It is running 90-seconds-long experiments in a loop with different loads.
+    * If you already know the saturation load, you can simply run the test (several times) for 90 seconds with a load somewhat
+      below saturation:
+        * set makefile variables `ROTATE_CONNECTIONS`, `ROTATE_TX_RATE`, to values that will produce the desired transaction load.
+        * set `ROTATE_TOTAL_TIME` to 90 (seconds).
+        * run "make runload" and wait for it to complete. You may want to run this several times.
+          so the data of different runs can be compared.
```
Comment (lasarojc) [inline docs/qa/method.md:77]:
```suggestion
        * run "make runload" and wait for it to complete. You may want to run this several times so the data from different runs can be compared.
```

File: docs/qa/method.md [hunk 3]
```diff
@@ -73,32 +93,37 @@ This section explains how the tests were carried out for reproducibility purpose
...
-The Core team should improve it at every iteration to increase the amount of automation.
+The CometBFT team should improve it at every iteration to increase the amount of automation.
 
 #### Steps
 
 1. Unzip the blockstore into a directory
 2. Extract the latency report and the raw latencies for all the experiments. Run these commands from the directory containing the blockstore
-    * `go run github.com/cometbft/cometbft/test/loadtime/cmd/report@3ec6e424d --database-type goleveldb --data-dir ./ > results/report.txt`
-    * `go run github.com/cometbft/cometbft/test/loadtime/cmd/report@3ec6e424d --database-type goleveldb --data-dir ./ --csv results/raw.csv`
+    * `go run github.com/cometbft/cometbft/test/loadtime/cmd/report@f1aaa436d --database-type goleveldb --data-dir ./ > results/report.txt`
```
Comment (lasarojc) [inline docs/qa/method.md:102]:
```suggestion
    * ```bash
       go run github.com/cometbft/cometbft/test/loadtime/cmd/report@f1aaa436d --database-type goleveldb --data-dir ./ > results/report.txt
       ```
```

Comment (lasarojc) [reply docs/qa/method.md:102]: it makes it easier to copy the command by just clicking on the icon top-left of the code.

File: docs/qa/method.md [hunk 4]
```diff
@@ -73,32 +93,37 @@ This section explains how the tests were carried out for reproducibility purpose
...
+    * `go run github.com/cometbft/cometbft/test/loadtime/cmd/report@f1aaa436d --database-type goleveldb --data-dir ./ --csv results/raw.csv`
```
Comment (lasarojc) [inline docs/qa/method.md:103]:
```suggestion
    * ```bash
       go run github.com/cometbft/cometbft/test/loadtime/cmd/report@f1aaa436d --database-type goleveldb --data-dir ./ --csv results/raw.csv
       ```
```

**Pull request merged**